src/main/java/plugily/projects/buildbattle/commands/arguments/admin/plot/SelectPlotArgument.java

```diff
@@ -38,12 +38,11 @@ public SelectPlotArgument(ArgumentsRegistry registry) {
     registry.mapArgument("buildbattle", new CommandArgument("selectplot", "buildbattle.command.selectplot", CommandArgument.ExecutorType.PLAYER) {
       @Override
       public void execute(CommandSender sender, String[] args) {
-        BaseArena arena = (BaseArena) registry.getPlugin().getArenaRegistry().getArena((Player) sender);
-        if(arena == null) {
-          return;
-        }
-        if(arena.getArenaState() == ArenaState.WAITING_FOR_PLAYERS || arena.getArenaState() == ArenaState.STARTING)
-          arena.getPlugin().getPlotMenuHandler().createMenu((Player) sender, arena);
+        Player player = (Player) sender;
+        BaseArena arena = (BaseArena) registry.getPlugin().getArenaRegistry().getArena(player);
+
+        if(arena != null && arena.getArenaState() == ArenaState.WAITING_FOR_PLAYERS || arena.getArenaState() == ArenaState.STARTING)
```

Comment:
making usage of https://github.com/Plugily-Projects/MiniGamesBox/blob/development/MiniGamesBox%20Classic/src/main/java/plugily/projects/minigamesbox/classic/arena/ArenaState.java#L51 isLobbyStage here

Reply:
The dependency of MinigamesBox needs to be updated in different PR, see #51 for further or I will make a PR for this.

Reply:
yep another pr would be good or may the author of #51 can add that (may these prs conflict with each other as I saw some same lines on #51 too)
As this pr was first we will try to merge this one first.